WalletWasabi.Gui/App.xaml

```diff
@@ -33,5 +35,50 @@
     <DataTemplate DataType="id:IView">
       <cont:ViewModelViewHost DataContext="{Binding Context}" />
     </DataTemplate>
+
+    <DataTemplate DataType="sys:Exception">
+      <TextBlock Text="{Binding Message}"/>
+    </DataTemplate>
+
+    <DataTemplate DataType="models:WarningMessageWrapper">
+      <StackPanel Orientation="Horizontal">
+        <DrawingPresenter Margin="2" VerticalAlignment="Center">
+          <DrawingGroup>
+            <GeometryDrawing Brush="#FFF6F6F6" Geometry="F1M7.5,2L2,12 4,14 13,14 15,12 9.5,2z" />
+            <GeometryDrawing Brush="#FFFFCC00" Geometry="F1M9,10L8,10 8,6 9,6z M9,12L8,12 8,11 9,11z M9,3L8,3 3.5,12 4.5,13 12.5,13 13.5,12z" />
+            <GeometryDrawing Brush="#FF000000" Geometry="F1M8,10L9,10 9,6 8,6z M8,12L9,12 9,10.999 8,10.999z" />
+          </DrawingGroup>
+        </DrawingPresenter>
+        <ContentControl Margin="2" VerticalAlignment="Center" Content="{Binding Message}"/>
+      </StackPanel>
+    </DataTemplate>
+
+    <DataTemplate DataType="models:ValidationMessageWrapper">
+      <StackPanel Orientation="Horizontal">
+        <DrawingPresenter Margin="2" VerticalAlignment="Center">
+          <DrawingGroup>
+            <GeometryDrawing Brush="#FFF6F6F6" Geometry="F1M7.5,2L2,12 4,14 13,14 15,12 9.5,2z" />
```

Comment:
These ugly Geometry icons will be moved to `Icons.xaml` when we can pick the right ones.